tests/net/lib/coap_client/CMakeLists.txt

```diff
@@ -24,6 +24,6 @@ add_compile_definitions(CONFIG_COAP_CLIENT_MESSAGE_HEADER_SIZE=48)
 add_compile_definitions(CONFIG_COAP_CLIENT_STACK_SIZE=1024)
 add_compile_definitions(CONFIG_COAP_CLIENT_THREAD_PRIORITY=10)
 add_compile_definitions(CONFIG_COAP_LOG_LEVEL=4)
-add_compile_definitions(CONFIG_COAP_INIT_ACK_TIMEOUT_MS=2000)
+add_compile_definitions(CONFIG_COAP_INIT_ACK_TIMEOUT_MS=200)
```

Comment:
I did not knew that you could do something like this. Usually the preferred way to modify the configuration is to add the options to testcase.yaml file. I just wonder why to add the options to cmake file.
Perhaps the coap and lwm2m tests could be changed to use the yaml file instead at some point.

Reply:
> I did not knew that you could do something like this. Usually the preferred way to modify the configuration is to add the options to testcase.yaml file. I just wonder why to add the options to cmake file. Perhaps the coap and lwm2m tests could be changed to use the yaml file instead at some point.
I'll add task to our backlog.

Reply:
Sometimes enabling the feature from Kconfig brings its dependencies too, which may make it harder to write module/unittests.
So some test cases are written so that they include just the `.c` file in `CMakeLists.txt` and add a definitions to Kconfig options that are not enabled, but required to build the file.